libraries/chain/include/golos/chain/database.hpp

```diff
@@ -72,6 +72,12 @@ namespace golos { namespace chain {
                 skip_database_locking = 1 << 15 ///< used to skip locking of database
             };
 
+            enum store_metadata_modes {
+                store_for_all,
+                store_for_listed,
+                store_for_no_one
```

Comment:
Rename to `store_metadata_for_XXX`.
`store_for_no_one` -> `store_metadata_for_nobody`